client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/staticstate/ObjectiveRecorder.java

```diff
@@ -83,7 +83,6 @@
      */
```

Comment:
why modifying this file??? or was it by mistake?

Reply:
Sorry it was by mistake